### DISCUSSION TITLE
Using getStaticProps to get Advanced Custom Fields data out of Wordpress

### DISCUSSION THREAD

**Discussion by ChrisHPZ** [vercel/next.js#57706] [Help] [answered]
### Summary

I've begun refactoring my headless setup to use getStaticProps. One of the things I'm having a difficult time with is in grabbing Advanced Custom Fields data and displaying it on screen. When I try to console.log() the data that should be grabbed, it comes back as undefined. I'm sure it's something simple that I'm overlooking, but I can't imagine what that might be.

### Additional information

```js
GraphQL Request code
https://gist.github.com/ChrisHPZ/386e51f4bd1d0d4eac0be7da8a031f58)

Advanced Custom Fields query
https://gist.github.com/ChrisHPZ/5f4f58229a37870598d7a47c3c1f2f84

Component where console.log fails
https://gist.github.com/ChrisHPZ/1d2fbaa620ca094a5d0362949c88cbc6
```

**Comment by icyJoseph** [comment, marked as answer]
How is `FooterContactRow` used? Do you have an `_app` file? Could you show that too?

In Pages Router `FooterContactRow` has to be a page level component, it cannot be nested. In other words, data fetching functions, such as `getStaticProps` can only be used at page level.

**Comment by ChrisHPZ** [comment]
Oh! That must be why this isn't working. `FooterContactRow` is a nested component.
